src/nyx/features/hexagonality_polygonality.h

```diff
@@ -23,5 +23,6 @@ class HexagonalityPolygonalityFeature: public FeatureMethod
 
 private:
 	double polyAve = 0, hexAve = 0, hexSd = 0;
+	static const int novalue = -1;	// feature falue made intentionally blank and unsuitable for model training

```

Comment:
We need to document it somewhere, I think.

Reply:
Later, when I do similar NAN intercept for other feature classes. I may have 3 more ones today